### PR TITLE
DRUP-753: #ready Displays the user's first and last name instead of drupal u…

### PR DESCRIPTION
**Pull request by z3cka**
…sername (aka. UID).

also, adds (UID) to the Username label on the user edit form

@akohler please pull to www-test along with another PR that will be coming in the morning regarding [DRUP-751](https://jira.library.ucla.edu/browse/DRUP-751).

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_users/uclalib_users.module [hunk 1]
```diff
@@ -5,3 +5,20 @@
...
 
 include_once 'uclalib_users.features.inc';
+
+/**
+ * Implements hook_username_alter().
+ */
+function uclalib_users_username_alter(&$name, $account) {
+  // Display the user's first and last name instead of drupal username (aka. UID).
+  if (property_exists($account, 'field_first_name') && property_exists($account, 'field_last_name')  ) {
+    $name = $account->field_first_name['und'][0]['value'] . '' . $account->field_last_name['und'][0]['value'];
```
Comment (akohler) [inline www/sites/all/modules/features/uclalib_users/uclalib_users.module]: @z3cka : Double-checking that you do *not* want a space between first and last names?  Looks like you're concatenating with an empty string '' and not a space ' '.

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_users/uclalib_users.module]: Good catch! I do indeed need a space.... will update PR to contein `' '`. Thanks!

**Comment by z3cka** [comment]
PR updated... can wait for deployment to `www-test` for DRUP-751

**Comment by akohler** [comment]
Merged; will wait for DRUP-751 PR before deploying to www-test, thanks.

**Pull request merged**